### PR TITLE
circle test

### PR DESCRIPTION
**Pull request by LoicGady-Exa**

### CODE REVIEW

**Comment by BasileTrujillo** [comment]
Try to use the V2 syntax and a docker container instead of using `machine`.
https://circleci.com/docs/2.0/

**Comment by BasileTrujillo** [comment]
can you also plug circle CI on this project ?

File: README.md [hunk 1]
```diff
@@ -1,22 +1,28 @@
 # MailHelper
+[![CircleCI](https://circleci.com/gh/exaprint/MailHelper.svg?style=shield)](https://circleci.com/gh/exaprint/MailHelper)
+[![Code Climate](https://codeclimate.com/github/exaprint/MailHelper/badges/gpa.svg)](https://codeclimate.com/github/exaprint/MailHelper)
+[![Test Coverage](https://codeclimate.com/github/exaprint/MailHelper/badges/coverage.svg)](https://codeclimate.com/github/exaprint/MailHelper/coverage)
```
Comment (BasileTrujillo) [inline README.md]: You can replace codeclimate by codeCov

File: phpUnit.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```
Comment (BasileTrujillo) [inline phpUnit.sh]: Remmove this file

Comment (LoicGady-Exa) [inline phpUnit.sh]: Done

File: README.md (hunk 1 above)
Comment (LoicGady-Exa) [inline README.md]: Done

**Pull request merged**